recipes-sota/aktualizr/aktualizr-ca-implicit-prov.bb

```diff
@@ -0,0 +1,67 @@
+SUMMARY = "Aktualizr configuration for implicit provisioning"
+DESCRIPTION = "Systemd service and configurations for implicitly provisioning Aktualizr, the SOTA Client application written in C++"
```

Comment:
Would you mind changing summary/description to say that this is the implicit provisioning with CA?